Dockerfile

```diff
@@ -52,7 +41,21 @@ RUN rm -rf /var/lib/apt/lists/* && \
         | tar -xz -C /opt/ants-2.3.4 --strip-components 1
 
 RUN pip install -U pip
-RUN cd /tractolearn && pip install -e . 
+
+RUN pip install git+https://github.com/scil-vital/tractolearn.git@{version} tractolearn
```

Comment:
The version should be pinned once it is tagged in `tractolearn`.

Reply:
see PR #4

Reply:
Tracked by #5. Resolving.